doc/ovs-hw-offload.md

```diff
@@ -0,0 +1,131 @@
+# OVS Hardware Offload
+
+The OVS software based solution is CPU intensive, affecting system performance
+and preventing fully utilizing available bandwidth. OVS 2.8 and above support
+a feature called OVS Hardware Offload which improves performance significantly.
+This feature allows offloading the OVS data-plane to the NIC while maintaining
+OVS control-plane unmodified. It is using SR-IOV technology with VF representor
+host net-device. The VF representor plays the same role as TAP devices
+in Para-Virtual (PV) setup. A packet sent through the VF representor on the host
+arrives to the VF, and a packet sent through the VF is received by its representor.
+
```

Comment:
we should have prerequisites section:
1. ovs installed 
2. Network Manager installed